rebaser.go

```diff
@@ -50,41 +56,64 @@ func (r *Rebaser) Rebase(workingImage imgutil.Image, newBaseImage imgutil.Image,
 	// get existing metadata label
 	var origMetadata files.LayersMetadataCompat
 	if err = image.DecodeLabel(workingImage, platform.LifecycleMetadataLabel, &origMetadata); err != nil {
-		return RebaseReport{}, errors.Wrap(err, "get image metadata")
+		return RebaseReport{}, fmt.Errorf("get image metadata: %w", err)
 	}
 
 	// rebase
 	if err = workingImage.Rebase(origMetadata.RunImage.TopLayer, newBaseImage); err != nil {
-		return RebaseReport{}, errors.Wrap(err, "rebase app image")
+		return RebaseReport{}, fmt.Errorf("rebase app image: %w", err)
 	}
 
 	// update metadata label
 	origMetadata.RunImage.TopLayer, err = newBaseImage.TopLayer()
 	if err != nil {
-		return RebaseReport{}, errors.Wrap(err, "get rebase run image top layer SHA")
+		return RebaseReport{}, fmt.Errorf("get rebase run image top layer SHA: %w", err)
 	}
 	identifier, err := newBaseImage.Identifier()
 	if err != nil {
-		return RebaseReport{}, errors.Wrap(err, "get run image id or digest")
+		return RebaseReport{}, fmt.Errorf("get run image id or digest: %w", err)
 	}
 	origMetadata.RunImage.Reference = identifier.String()
-	if appPlatformAPI != "" && api.MustParse(appPlatformAPI).AtLeast("0.12") {
+	if appPlatformAPI != "" && r.PlatformAPI.AtLeast("0.12") {
```

Comment:
Should we still be checking for the presences of `appPlatformAPI` in order to continue?

Reply:
Oh whoops... thank you!

Reply:
Updated in f29ebbb6